bin/src/Main.purs

```diff
@@ -536,6 +554,10 @@ main = do
             env' <- runSpago env (mkBuildEnv buildArgs dependencies)
             let options = { depsOnly: true, pursArgs: List.toUnfoldable args.pursArgs, jsonErrors: false }
             runSpago env' (Build.run options)
+          Uninstall { packagesToRemove, selectedPackage, testDeps } -> do
+            { env, fetchOpts } <- mkFetchEnv offline { packages: packagesToRemove, selectedPackage, ensureRanges: false, testDeps: false }
```

Comment:
The `packages` here are additional packages to add to the config, which is the opposite of what we want to achieve here 😄 
```suggestion
            { env, fetchOpts } <- mkFetchEnv offline { packages: [], selectedPackage, ensureRanges: false, testDeps: false }
```

Reply:
> ...are additional packages to add to the config

AFAICT via [`mkFetchEnv`](https://github.com/purescript/spago/blob/master/bin/src/Main.purs#L880-L897), that's not true. Rather, we just validate there that the strings we got from the CLI can be parsed into `PackageName` values. The resulting `packageNames` value is never used anywhere else and is just stored in `fetchOpts`. And for context, the only reason why I'm using `mkFetchEnv` instead of doing this outside of that is because I need a `workspace` value.

Reply:
Ah yeah sorry - I got confused about `mkFetchEnv` vs `Fetch.run`, all good